CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ## Unreleased
 
 - Add remove button to link tooltip
+- Fix tabbing through the toolbar in Firefox - PR [here](https://github.com/alphagov/govspeak-visual-editor/pull/123)
```

Comment:
Minor nitpick, since it's documentation, but avoid linking just the word "here" as it's inaccessible for screenreader users.

```suggestion
- Fix tabbing through the toolbar in Firefox - [PR #123](https://github.com/alphagov/govspeak-visual-editor/pull/123)
```